src/game/mario_actions_automatic.c

```diff
@@ -565,11 +565,9 @@ s32 act_ledge_grab(struct MarioState *m) {
     if (m->actionTimer < 10) {
         m->actionTimer++;
     }
-#ifdef LEDGE_GRABS_CHECK_SLOPE_ANGLE
```

Comment:
todo here, this causes slope stutters which sucks. ig this should check if the floor is slippery too

Reply:
well regardless this is the exact same check as the one i replaced it with so it wont really behave too differently to how it was already, still a direct upgrade